README.md

```diff
@@ -246,55 +282,111 @@ plugin, please consider one of the following options:
 - To use a config file located outside the package/project directory, a config 
   file may be added to that directory specifying a parent config path to the 
   other config file, e.g., `parent_config: path/to/.swiftlint.yml`.
-- You can also consider the use of a Run Script Build Phase in place of the
-  build tool plugin.
+- You can also consider the use of a
+  [Run Script Build Phase](#xcode-run-script-build-phase) in place of the build
+  tool plugin.
+
+### Xcode Run Script Build Phase
+
+If the build tool plugin does not work for your project setup or when
+additional custom setup is required, SwiftLint can be added as a Run Script
+Build Phase. This is useful when a project setup relies on the `--config`
+SwiftLint option or to lint all targets together in a single `swiftlint`
+invocation.
+
+> [!IMPORTANT]
+> Compatibility is dependent upon the [installation](#installation) method that
+is used.
+
+To do this, add a custom script to a `Run Script` phase of the `Build Phases`
+of the primary app target, before the `Compile Sources` phase. Use the
```

Comment:
Based on this language in another part of the readme, it sounds like there are pros/cons to linting before or after compiling. Since this is a general concern (not specific to any installation method for example), I wonder if this language could be improved and moved to a dedicated section between the Installation and Setup sections?

> You might want to move your SwiftLint phase directly before the 'Compile Sources' 
> step to detect errors quickly before compiling. However, SwiftLint is designed to 
> run on valid Swift code that cleanly completes the compiler's parsing stage. So 
> running SwiftLint before 'Compile Sources' might yield some incorrect results.

Reply:
What about this?
> While it may seem intuitive to run SwiftLint before compiling Swift source 
> files to exit a build early when there are lint violations, it is important 
> to understand that SwiftLint is designed to analyze valid source code that 
> is compilable. Non-compiling code can very easily lead to unexpected and 
> confusing  results, especially when executing with `--fix`/`--autocorrect` 
> command line arguments.

Reply:
Sounds good. Could be an IMPORTANT box in the Setup section.